### PR TITLE
[SIMPLE] readability: reduce level of indents for branch.init()

### PR DESCRIPTION
**Pull request by Kirkkt**
I'm about to make some non-trivial changes but found this function a pain to read/edit. Essentially an NOP.

@dmitrig01

### CODE REVIEW

**Comment by Kirkkt** [comment]
Might as well do this across the board and set a lint rule for it :)

**Comment by Kirkkt** [comment]
friendly ping
@dmitrig01

File: src/3_branch.js [hunk 1]
```diff
@@ -446,82 +446,84 @@ Branch.prototype['init'] = wrap(
...
+			return;
 		}
-		else {
-			if (CORDOVA_BUILD || TITANIUM_BUILD) {
 
-				var apiCordovaTitanium = function(data) {
-					if (!freshInstall) {
-						data['identity_id'] = sessionData['identity_id'];
-						data['device_fingerprint_id'] = sessionData['device_fingerprint_id'];
+		if (WEB_BUILD) {
```
Comment (dmitrig01) [inline src/3_branch.js]: i'm pretty sure we still need an else, no? we can definitely turn it into an elseif though

Comment (Kirkkt) [inline src/3_branch.js]: it returns in the bottom

Comment (dmitrig01) [inline src/3_branch.js]: ah ok cool

**Comment by dmitrig01** [comment]
:+1:

**Comment by dmitrig01** [comment]
actually hang on

**Comment by dmitrig01** [comment]
you sure if resources.queryPart is undefined that loop won't throw an error (in all browsers)?

**Comment by Kirkkt** [comment]
@dmitrig01 Great point! I'm not 100% sure so it's better safe than sorry.

Thank you!

**Comment by dmitrig01** [comment]
:+1:

if you can check this, we do this in a lot of places :-)

**Comment by Kirkkt** [comment]
Done :)

**Comment by dmitrig01** [comment]
nice :+1:

**Pull request merged**